employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java

```diff
@@ -19,4 +22,14 @@
       return new ResponseEntity<>("Employee created successfully",HttpStatus.CREATED);
     }
 
+    @PostMapping("/createEmployeeList")
```

Comment:
I would call this as createEmployees. If you change it please change the name of method too

Reply:
Changed as requested